lua/nvim-treesitter/parsers.lua

```diff
@@ -85,6 +85,15 @@ list.cuda = {
   maintainers = { "@theHamsta" },
 }
 
+list.d = {
+  install_info = {
+    url = "https://github.com/CyberShadow/tree-sitter-d",
+    files = { "src/parser.c", "src/scanner.cc" },
+    requires_generate_from_grammar = true,
+  },
+  maintainers = { "@CyberShadow" },
```

Comment:
It will not be maintained when you remove this line here. You could also add a configurable table that removes certain parsers in `install.lua` from `maintained`

Reply:
Is it actually a list of parser maintainers or maintainers of the integrations in the repo? `@CyberShadow` is a maintainer of the parser, it is not me. I just wanted to add an integration here

Reply:
It's the maintainers of the highlights and the integration into nvim-treesitter.